### DISCUSSION TITLE
get code versions for a set of assets being materialized

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#15197] [Q&A]
Hey folks, is it possible to get a reference to the assets_definition of all of the other assets being materialized in a particular dagster run from within an asset? i.e. via the context object? I'd like to get the asset code_version for all assets being materialized.

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1689002667735029?thread_ts=1689002667.735029&cid=C01U954MEER).*

**Comment by smackesey** [comment]
This is not currently possible via public APIs. Here is a tracking issue for it: https://github.com/dagster-io/dagster/issues/15283
